crates/bevy_gilrs/src/lib.rs

```diff
@@ -20,10 +23,13 @@ impl Plugin for GilrsPlugin {
         {
             Ok(gilrs) => {
                 app.insert_non_send_resource(gilrs)
+                    .add_event::<RumbleRequest>()
+                    .init_non_send_resource::<rumble::RumblesManager>()
```

Comment:
```suggestion
                    .init_non_send_resource::<RumblesManager>()
```

This should just be imported.